src/states/index.js

```diff
@@ -0,0 +1,19 @@
+const BOOKS = [
```

Comment:
Oops! according to the instructions on pathwright we need to create the books in `src/index.js`
![image](https://user-images.githubusercontent.com/31897434/124326033-aa54a100-db7d-11eb-82e4-4d5df6638654.png)